Implementation/Services/index.md

```diff
@@ -0,0 +1,510 @@
+    public static class UmbracoHelperExtensions
+    {
+            //NewsCategories //NewsSection
+            public static IPublishedContent GetNewsSection(this UmbracoHelper umbracoHelper)
+            {
+            return umbracoHelper.ContentSingleAtXPath("root/homePage/newsSection");
+            }
+     }
+}
+```
+
+anywhere there is reference to the UmbracoHelper, and a reference is added to the namespace the extension belongs to, it is possible to call the method by writing `Umbraco.GetNewsSection()`
+
+### Custom Services and Helpers
+
+Another option, is to make use of the underlying DI framework, and create custom Services and Helpers, that in turn can have the 'core' Management Services and Umbraco Helpers injected into them - (the same rules about transient services apply!). This approach enables the grouping together of similar methods within a suitably named service, and promotes the possibility of testing this custom logic outside of controllers and views.
+
+In this example, we create a custom helper service, that's responsible for finding key pages within a site, eg the News Section or the Contact Us page - these methods will commonly be called in different places throughout the site, and it's great to encapsulate the logic to retrieve them in a single place - we'll call this helper service: SiteHelperService
+
+Create an interface to define the service:
+
+```csharp
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Web;
+using Umbraco.Core.Models.PublishedContent;
+
+namespace Umbraco8.Services
+{
+        public interface ISiteHelperService
+    {
+        IPublishedContent GetNewsSection();
+        IPublishedContent GetContactUsPage();
+    }
+}
+```
+Create the concrete service class that implements the interface:
+```csharp
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Web;
+using Umbraco.Core.Models.PublishedContent;
+
+namespace Umbraco8.Services
+{
+    public class SiteHelperService : ISiteHelperService
+    {
+        public IPublishedContent GetNewsSection()
+        {
+            //ToDo: implement this!
+            throw new NotImplementedException();
+        }
+        public IPublishedContent GetContactUsPage()
+        {
+            //ToDo: implement this!
+            throw new NotImplementedException();
+        }
+    }
+}
+```
+
+Register the custom service with Umbraco's underlying DI container using an IUserComposer:
+
+```csharp
+using Umbraco.Core;
+using Umbraco.Core.Composing;
+using Umbraco8.Services;
+
+namespace Umbraco8.Composers
+{
+    public class RegisterSiteHelperServiceComposer : IUserComposer
+    {
+        public void Compose(Composition composition)
+        {         
+            composition.Register<ISiteHelperService, SiteHelperService>(Lifetime.Request);          
+        }
+    }
+}
+```
+
+Note we register the service in 'Lifetime.Request' scope - as we'll be making use of the transient UmbracoContext
```

Comment:
It is **not** Transient, it is also Request lifetime. 

I have another note below about how to structure these types of things, registering things as Lifetime.Request should be avoided wherever possible and perhaps we make a note of that.

Reply:
Updated, and the Lifetime info added